sdk/ios/Configsum/Configsum/Configsum/Networking/Router.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Router.swift
+//  Configsum
+//
+//  Created by Alexandru Savu on 2017-10-04.
+//  Copyright © 2017 Lifesum. All rights reserved.
+//
+
+import Foundation
+
+internal struct Router {
+    //TODO Remove hardcoded elements
+    fileprivate struct Constants {
+        static let authorization = "appTokenApp12345"
+        static let userAuthorizationToken = "userAuthToken54321"
+        static let tokenHeader = "X-Configsum-Token"
+        static let authorizationKey = "Authorization"
+        static let contentType = "application/json"
+        static let contentTypeKey = "Content-Type"
+    }
+    
+    private let environment: Environment
+    private var requestURL: URL? {
+        let components = NSURLComponents()
+        components.scheme = "http"
+        components.host = "localhost"
```

Comment:
can we extract the url components from the environment? hardcoding host & port is problematic 

Something like `Enviornment(Debug)` & `Enviornment(Production)`